### PR TITLE
Support for PHP 8

### PR DESCRIPTION
**Pull request by lstrojny**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
         }
     ],
     "require": {
-        "php": "^5.3 || ^7.0"
+        "php": "^5.3 || ^7.0 || ^8.0"
```
Comment (willdurand) [inline composer.json:13]: Ha, that was easy! Is it all?

I probably should update the CI infra too...

Comment (stof) [reply composer.json:13]: for the CI, this will require updating to newer PHPUnit versions. The old PHPUnit 5.7 does not support PHP 8

Comment (willdurand) [reply composer.json:13]: Yeah, I suppose we can use the symfony phpunit "bridge" (or whatever it's called), right?

Comment (stof) [reply composer.json:13]: Well, using composer is also fine.

But in both cases, you will need to adapt the testsuite to be compatible with newer PHPUnit versions. Note that dropping support for older PHP versions might help a bit here, by reducing the range of PHPUnit versions you need to support. Supporting PHPUnit 4 and 8 at the same time is quite painful. See https://phpunit.de/supported-versions.html for the supported PHP versions for each PHPUnit major version.

**Comment by willdurand** [comment]
I changed the constraint recently.

**Pull request closed without merging**